packages/algob/src/lib/lsig.ts

```diff
@@ -17,3 +18,22 @@ export async function getLsig (name: string, scParams: Object, algodClient: algo
   const program = result.toBytes;
   return algosdk.makeLogicSig(program, scParams);
 }
+
+/**
+ * Description: this function decodes msig object from logic signature
+ * @param {string} msig : multisigned msig obj
+ * @returns {Object} : decoded msig (object with decoded public keys and their signatures)
+ */
+export async function decodeMsigObj (msig: string): Promise<Object> {
```

Comment:
Please use a proper return type, not `Object`. the `parsedMsig` must have some interface.

Reply:
added interface for `MultiSig` in types.